src/Verifier/LLVM/Codebase.hs

```diff
@@ -199,7 +199,7 @@ loadModule bcFile = do
     Right mdl -> return mdl
  where
     parse = BS.readFile >=> BC.parseBitCode
-    err msg = fail $ "Bitcode parsing of " ++ bcFile ++ " failed:\n"
+    err msg = error $ "Bitcode parsing of " ++ bcFile ++ " failed:\n"
```

Comment:
nit: is it possible to throw an exception here instead of `error`? This function is expected to fail when the file is not a valid bitcode file.